Adding docs for setDelegationParameters() (#548)
graphprotocol/docs: website/pages/en/network/indexing.mdx

setDelegationParameters(950000, 600000, 500)
```

### Setting delegation parameters

The `setDelegationParameters()` function in the [staking contract](https://github.com/graphprotocol/contracts/blob/main/contracts/staking/Staking.sol) is essential for Indexers, allowing them to set parameters that define their interactions with Delegators, influencing their reward sharing and delegation capacity.

### How to set delegation parameters

To set the delegation parameters using the Graph Explorer interface, follow these steps:

1. Navigate to [The Graph Explorer](https://thegraph.com/explorer/).

> 2. Connect your wallet. Choose multi-sig (such as Gnosis Safe) and then select mainnet. Note: You will need to repeat this process for Arbitrum One.

Reviewer: ```suggestion
2. Connect your wallet. Choose multisig (such as Gnosis Safe) and then select mainnet. Note: You will need to repeat this process for Arbitrum One.
```

setDelegationParameters(950000, 600000, 500)
```

### Setting delegation parameters

> The setDelegationParameters() function in the [staking contract](https://github.com/graphprotocol/contracts/blob/main/contracts/staking/Staking.sol) is essential for Indexers, allowing Indexers to set parameters that define their interactions with Delegators, influencing their reward sharing and delegation capacity.

Reviewer: ```suggestion
The `setDelegationParameters()` function in the [staking contract](https://github.com/graphprotocol/contracts/blob/main/contracts/staking/Staking.sol) is essential for Indexers, allowing them to set parameters that define their interactions with Delegators, influencing their reward sharing and delegation capacity.
```

This function enables an Indexer to define the following parameters:

> - indexingRewardCut: Percentage of indexing rewards retained for the Indexer.
> - queryFeeCut: Percentage of query fees retained by the Indexer.
> - \_cooldownBlocks: Period of time (in blocks) that need to pass to update delegation parameters.

Reviewer: ```suggestion
- `indexingRewardCut`: Percentage of indexing rewards retained for the Indexer.
- `queryFeeCut`: Percentage of query fees retained by the Indexer.
- `_cooldownBlocks`: Period of time (in blocks) that need to pass to update delegation parameters.
```

> - `_cooldownBlocks`: Period of time (in blocks) that need to pass to update delegation parameters.

Reviewer 2: I think this is being changed soon @pcarranzav?

Reviewer 3: Yes, the Snapshot is up for the Council to vote, and we should be deploying this relatively soon

### How to set delegation parameters

> To set the delegation parameters using The Graphs Explorer interface, follow these steps:

Reviewer: ```suggestion
To set the delegation parameters using the Graph Explorer interface, follow these steps:
```

1. Navigate to [The Graph Explorer](https://thegraph.com/explorer/).

> 2. Connect your wallet. Choose Gnosis Safe and then select Mainnet. Note: You will need to repeat this process for Arbitrum One.

Reviewer 2: Why always a Gnosis safe?

3. Connect the wallet you have as a signer.

> 4. Navigate to the 'Settings' section and select 'Delegation Parameters'. These parameters should be configured to achieve an effective cut within the range of approximately X-Y%. Upon entering values in the provided input fields, the interface will automatically calculate the effective cut. Adjust these values as necessary to attain the desired effective cut percentage.

Reviewer 2: X-Y%? Is this intentionally a placeholder?

Author: Actually a typo - thank you for catching!
